docs/source/standard.md

```diff
@@ -223,33 +223,87 @@ The ability of the TRE organisation to manage data assets and ensure information
 
 The ability of the TRE organisation to manage how and where data is stored, how it moves, changes and is removed.
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+```{list-table}
+:header-rows: 1
+:name: tab-data-lifecycle-management
+
+* - Statement
+  - Guidance
+* - A TRE must have a data ingress process which enforces information governance rules/processes.
+  - The data ingress process needs to ensure that information governance is correctly followed.
+    in particular, it should require that an ingress request has been approved by all required par ties.
+* - A TRE must have a data egress process which enforces information governance rules/processes.
+  - The data egress process needs to ensure that information governance requirements are adhered to.
+    In particular, it should require that an egress request has been approved by all required parties.
+* - A TRE could require independent approval for data ingress and egress events.
+  - A person independent to a project, for example a TRE admin or dedicated referee, could check ingress and egress requests to ensure information governance processes are correctly followed in the absence of an automated process.
+* - A TRE must keep a record of what data it holds.
+  - Good records are important for ensuring compliance with legislation, understanding risk and aiding good data hygiene.
+    The record should include a description of the data, its source, contact details for the data owner, which projects use the data, the data it was recieved, when it is expected to no longer be needed.
```

Comment:
```suggestion
    The record should include a description of the data, its source, contact details for the data owner, which projects use the data, the date it was recieved, when it is expected to no longer be needed.
```
May need to update this in future to include streaming data, or datasets that are regularly updated.